res/TensorFlowLiteRecipes/MaxPoolWithArgmax_002/test.recipe

```diff
@@ -1,20 +1,20 @@
 operand {
   name: "ifm"
   type: FLOAT32
-  shape { dim: 1 dim: 18 dim: 18 dim: 1 }
+  shape { dim: 1 dim: 18 dim: 18 dim: 2 }
 }
 operand {
   name: "ofm"
   type: FLOAT32
-  shape { dim: 1 dim: 8 dim: 8 dim: 1 }
+  shape { dim: 1 dim: 8 dim: 8 dim: 2 }
 }
 operand {
   name: "argmax"
   type: INT64
-  shape { dim: 1 dim: 8 dim: 8 dim: 1 }
+  shape { dim: 1 dim: 8 dim: 8 dim: 2 }
```

Comment:
I changed number of channels to diversify this test.
Otherwise all three tests had one common set of check rules.